lib/transaction.js

```diff
@@ -256,13 +228,17 @@ function makeTransactor(trx, connection, trxClient) {
   transactor.isTransaction = true;
   transactor.userParams = trx.userParams || {};
 
-  transactor.context.transaction = function(container, options) {
-    if (!options) {
-      options = { doNotRejectOnRollback: true };
-    } else if (isUndefined(options.doNotRejectOnRollback)) {
+  transactor.context.transaction = function(container, _config) {
+    const options = Object.assign({}, _config);
```

Comment:
Since we are using fairly new Node version, `const options = {..._config}` seems to be a simpler way.